app/assets/stylesheets/task.scss

```diff
@@ -22,4 +22,9 @@
 
 .need__created{
     background: #F7C745;
+}
+
+a.reset-needs-filters{
```

Comment:
Can we move this to [inside here](https://github.com/wearefuturegov/beacon/blob/master/app/assets/stylesheets/data_tables.scss#L117)? `task.scss` appears to be styles for viewing a need; the styles for the filters control on the needs list are in `data_tables.scss`.

Reply:
done